security/pkg/server/ca/authenticate/kube_jwt_test.go

```diff
@@ -69,26 +69,6 @@ func TestAuthenticate(t *testing.T) {
 			},
 			expectedErrMsg: "target JWT extraction error: no bearer token exists in HTTP authorization header",
 		},
-		"Review error": {
```

Comment:
Why are these tests removed?

Reply:
They test 'invalid JWT policy' - the change is replacing that, we no longer use the JWT policy.